03/app.js

```diff
@@ -0,0 +1,52 @@
+// get .images-list
+let imagesList = document.querySelector('.images-list');
+let input = document.querySelector('input');
+
+console.log(imagesList);
+console.log(input);
+
+// event listener for _change event_
+input.addEventListener('change', function(el) {
+  // CHANGE event works properly
+
+  // loop over all elements
+  for (let i = 0; i < el.target.files.length; i++) {
+
+    const file = el.target.files[i];
+
+    // clone `.images-list__item--prototype`
+    let newElement = document.querySelector('.images-list__item').cloneNode(true);
+    // remove proto class
+    newElement.classList.remove('images-list__item--prototype');
+
+    // check if file type is _image_
+    if (file && file.type.includes('image')) {
+
+      const read = new FileReader();
+      read.onload = function(reader) {
+        const addImg = document.createElement('img'); // is addImg = new Image() the same?
```

Comment:
Niby tak, ale nie do końca :D
Znalazłem takie info: https://stackoverflow.com/questions/6241716/is-there-a-difference-between-new-image-and-document-createelementimg